next.config.js

```diff
@@ -1,7 +1,10 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
-  reactStrictMode: true,
-  swcMinify: true,
+    reactStrictMode: true,
+    swcMinify: true,
+    eslint: {
+        dirs: ['pages', 'components', 'api']
```

Comment:
We should add `common` here too. I forgot to do it in my PR

Reply:
Done :)